packages/core/src/sagas/sendMessageBackToPostActivitySaga.js

```diff
@@ -1,10 +1,10 @@
 import { put, takeEvery } from 'redux-saga/effects';
-
-import { SEND_MESSAGE_BACK } from '../actions/sendMessageBack';
 import postActivity from '../actions/postActivity';
+import { SEND_MESSAGE_BACK } from '../actions/sendMessageBack';
```

Comment:
It's okay if you prefer this way. Just make sure to commit the Prettier config, so everyone can just use the "format document" command in VSCode to get consistent result.